docs/whats_new.md

```diff
@@ -577,6 +577,9 @@ In this example, the `Bind` method is used to bind the request body to the `User
 
 Fiber v3 enhances the redirect functionality by introducing new methods and improving existing ones. The new redirect methods provide more flexibility and control over the redirection process.
 
+The default response status is changes from 302 Found to 303 See Other for more
```

Comment:
This statement needs work. It should be a separate section.

Reply:
@andradei can you improve this